sqlup-mode.el

```diff
@@ -56,7 +56,8 @@
 
 ;;; Code:
 
-(require 'cl)
+(require 'cl-lib)
```

Comment:
The things in `cl` aren't namespaced (e.g. it was `some` instead of `cl-some` before) so it's much more likely that loading the library will overwrite user defined functions/variables.

I can't find the proper documentation, but there's a bit more information [here](https://www.emacswiki.org/emacs/CommonLispForEmacs).